.gitignore

```diff
@@ -281,3 +281,4 @@ config.ps1
 
 # VS debug support files
 launchSettings.json
+/src/Microsoft.VisualBasic/tests/VB/FileCleanupTestBase.vb
```

Comment:
Is this entry necessary?

Reply:
I excluded the file from my project.  I may need some help removing it from GitHub if that is what you are asking.